noxfile.py

```diff
@@ -27,6 +27,7 @@ def test(session):
         "pytest",
         "--cov=installer",
         "--cov-fail-under=100",
+        "--cov-report=term-missing",
```

Comment:
I usually like to also generate the HTML report with context to ``{envtmpdir}`` 👍

Reply:
Do you happen to recall the argument to use? I don’t use that often and it’s a hassle to dig into documentation.

Reply:
``
--cov-report=html:{toxworkdir}/htmlcov \
--cov-context=test \
```

Reply:
Thanks!